objectivec/cxx_api.h

```diff
@@ -11,28 +11,28 @@
 #endif  // defined(__clang__)
 
 // paths are different when building the Swift Package Manager package as the headers come from the iOS pod archive
+#define STRINGIFY(x) #x
 #ifdef SPM_BUILD
-#define HEADER_DIR "onnxruntime/"
+#define HEADER_FILE_PATH(x) STRINGIFY(onnxruntime / x)
```

Comment:
does this add spaces in the middle?

this build will test the SPM_BUILD path:
https://aiinfra.visualstudio.com/Lotus/_build/results?buildId=315012&view=results

I think we may need to exclude some additional files in Package.swift:
https://github.com/microsoft/onnxruntime/blob/2e66bc8669d56d7730a9da0a93e97711867db03c/Package.swift#L35

Reply:
hm, the ios packaging pipeline failed. looks like we will need to also fix up the cocoapods package setup too, as it unconditionally includes the training files now. initially we can just exclude them.

Reply:
Temporary added preprocessor guards to fix cocoapods package. 

[https://aiinfra.visualstudio.com/Lotus/_build/results?buildId=315350&view=results](https://aiinfra.visualstudio.com/Lotus/_build/results?buildId=315350&view=results)